Source/Core/Core/FreeLookManager.cpp

```diff
@@ -0,0 +1,232 @@
+// Copyright 2020 Dolphin Emulator Project
+// Licensed under GPLv2+
+// Refer to the license.txt file included.
+
+#include "Core/FreeLookManager.h"
+
+#include "Common/Common.h"
+#include "Common/CommonTypes.h"
+#include "Common/Config/Config.h"
+
+#include "Core/ConfigManager.h"
+#include "Core/FreeLookConfig.h"
+
+#include "InputCommon/ControllerEmu/ControlGroup/Buttons.h"
+#include "InputCommon/InputConfig.h"
+
+#include "VideoCommon/FreeLookCamera.h"
+#include "VideoCommon/OnScreenDisplay.h"
+
+namespace
+{
+enum MoveButtons
+{
+  Up,
+  Down,
+  Left,
+  Right,
+  Forward,
+  Backward
```

Comment:
This should have a trailing comma to reduce diff noise if more buttons are added in the future.